transform_encoder.py

```diff
@@ -0,0 +1,131 @@
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+from typing import Dict, Generator, Optional, Tuple
+
+import numpy as np
+import torch
+from jina import DocumentArray, Executor, requests
+from jina.logging.predefined import default_logger as logger
+from transformers import AutoModel, AutoTokenizer
+
+
+class TransformerTorchEncoder(Executor):
+    """
+    The transformer torch encoder encodes sentences into embeddings.
+    """
+
+    def __init__(
+        self,
+        pretrained_model_name_or_path: str = "sentence-transformers/distilbert-base-nli-stsb-mean-tokens",
+        base_tokenizer_model: Optional[str] = None,
+        pooling_strategy: str = "mean",
+        layer_index: int = -1,
+        max_length: Optional[int] = None,
+        embedding_fn_name: str = "__call__",
+        device: str = "cpu",
+        default_traversal_path: str = "r",
```

Comment:
traversal paths is a `List[str]`

Reply:
If we give multiple traversal paths, should we then compute the embedding for all of them?

Reply:
https://github.com/jina-ai/executor-transformer-torch-encoder/pull/1/commits/36d59f89cb704539bfba0fc7254fd2bfe0b9e9b2